### PR TITLE
feat(config): new configuration for ad370 device

### PR DESCRIPTION
**Pull request by kimjohnsson**

### CODE REVIEW

**Comment by kimjohnsson** [comment]
The z-wave alliance doesn't specify a Product Type ID for this device.. How do I handle that? I put the `productType` same as `productId` but not sure this is right...

**Comment by AlCalzone** [comment]
If you use Z-Wave JS UI, it should show you the full ID when you open the node "dropdown".

File: packages/config/config/devices/0x0060/ad370.json [hunk 1]
```diff
@@ -0,0 +1,209 @@
+{
+	"manufacturer": "Everspring",
+	"manufacturerId": "0x0060",
+	"label": "AD370",
+	"description": "Z-Wave 700 MOS Dimmer",
+	"devices": [
+		{
+			"productType": "0x0033",
+			"productId": "0x0033",
+			"zwaveAllianceId": [4514, 4648]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0060/ad370.json:10]: According to the ZWA website, these are two different manufacturers (Everspring and NEXA). Do you know if they use the same IDs? Often those re-branded devices end up with a different manufacturer ID or so.

File: packages/config/config/devices/0x0060/ad370.json [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Switch Type",
+			"description": "Def 0",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0060/ad370.json:21]: Unnnecessary description.

File: packages/config/config/devices/0x0060/ad370.json [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true
+		},
+		{
+			"#": "2",
+			"label": "Auto-on time",
+			"description": "Def 0, second",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0060/ad370.json:31]: Unnnecessary description. Units should be provided via the `unit` property.

File: packages/config/config/devices/0x0060/ad370.json [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+			"valueSize": 2,
+			"minValue": 1,
+			"maxValue": 100,
+			"defaultValue": 10,
+			"unsigned": true
+		},
+		{
+			"#": "9",
+			"label": "MOS operation mode",
+			"description": "Def 2",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0060/ad370.json:101]: Is there an explanation for the values? Those should probably be `options`.

File: packages/config/config/devices/0x0060/ad370.json [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 2,
+			"unsigned": true
+		},
+		{
+			"#": "16",
+			"label": "Wire connection type",
+			"description": "0: unknow, 1:2 wire, 2:3 wire",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0060/ad370.json:171]: This should be converted to options.

**Review by AlCalzone — CHANGES_REQUESTED** [review]
This does not match our style guide, especially regarding labels and descriptions (which we avoid in favor of more useful properties like unit, options, ...).

Please give https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide?id=labels (and the following sections) a good read and apply it throughout.

**Pull request closed without merging**